src/MANYBODY/pair_eam.cpp

```diff
@@ -43,6 +43,7 @@ PairEAM::PairEAM(LAMMPS *lmp) : Pair(lmp)
   nmax = 0;
   rho = NULL;
   fp = NULL;
+  count_embed = NULL;
```

Comment:
looks good - except let's call this numforce[],
similar to numneigh[], since it is the number
of atoms within the force cutoff

Reply:
done